app/services/fetch_support_rotations.rb

```diff
@@ -0,0 +1,38 @@
+require 'support_rota_client'
+
+class FetchSupportRotations
+  def initialize(client: SupportRotaClient.new)
+    @client = client
+  end
+
+  attr_accessor :client
+
+  def call
+    SupportDay::SUPPORT_TYPES.each do |support_type|
```

Comment:
Feels like the support types belong here, not in `SupportDay`, as they're specific to the service we're fetching from.

Reply:
Eh, not necessarily? They're specific to dxw, really, not to OpsGenie. The specific ways in which we at dxw define dev support, ops support, and the two OoHs.

Reply:
🤷

Reply:
Aargh, I've just realised that we'll have the same problem with support days that we had with [support assignments](https://github.com/dxw/team-dashboard/pull/51): we're never deleting support days, so if the rota gets adjusted (which I expect it to be, especially for the future), the new days will overlap with the old, inaccurate days. /o\

Reply:
OK, I've added some code to wipe out existing support days before importing the latest data, and reworded a couple of commit messages.

Do you have time to give it a (hopefully) final look and yay or nay it? @erbridge